lib/tasks/spec.rake

```diff
@@ -0,0 +1,12 @@
+require 'colorize'
+
+Rake::Task[:spec].clear
+
+desc "Run all spec"
+task :spec do
+  Rake::Task['spec:controllers'].invoke
```

Comment:
What is `spec:controller`?

Reply:
spec:controllers and spec:requests are commands for running tests for the controllers(endpoints, unit tests) and requests(integration tests). I could treat everything in the spec directory as unit tests and have a seperate directory (functional) just for the functional/integration tests. would that help clearing things up?